elrond_dex_farm/src/lib.rs

```diff
@@ -561,7 +570,7 @@ pub trait Farm {
         })
     }
 
-    #[view(simulateExitFarm)]
+    #[endpoint(simulateExitFarm)]
```

Comment:
simulateExitFarm should only calculate the minted rewards and not actually mint them